docs/development.rst

```diff
@@ -152,26 +152,107 @@ The top-level document to open with a web browser will be
 Testing
 -------
 
-To run unit tests in the currently active Python environment, issue one of
-these example commands:
+All of the following `make` commands run the tests in the currently active
+Python environment. Depending on how the `pywbem` package is installed in that
+Python environment, either the directories in the main repository directory
+are used, or the installed package.
+
+The test case files and any utility functions they use are always used from
+the main repository directory, and are located in the `testsuite` directory
+structure:
 
 ::
 
-    $ make test                                              # Run all unit tests
-    $ PYTHONPATH=. py.test testsuite/test_cim_obj.py -s      # Run only this test source file
-    $ PYTHONPATH=. py.test InitCIMInstanceName -s            # Run only this test class
-    $ PYTHONPATH=. py.test -k InitCIMInstanceName or Bla -s  # py.test -k expressions are possible
+    testsuite
```

Comment:
Why not go whole hog here and change testsuite to tests also

Reply:
That would bring pywbem into line with a lot of other python projects.

Reply:
Sounds good to me. DONE.